Makefile

```diff
@@ -66,10 +66,10 @@ env/bin/python:
 .PHONY: install
 install:
 	python3 --version
-	python3 -m pip install . --quiet
+	python3 -m pip install .[tests] --quiet
```

Comment:
I think we should leave this as it is because `make install` can be used to more thank just running test. So we don't want to install test dependencies when we just want to build `xml2rfc`.

`python3 -m pip .[tests] --quiet` should go under the `test:` in line 72.

Reply:
Needed to create `installtestdeps` rather than just adding `python3 -m pip .[tests] --quiet`
Probably more of the commands in the Makefile should be indicated as .PHONY